assets/src/js/dokan-frontend.js

```diff
@@ -53,12 +53,22 @@
     },
 
     init_category_widget_css() {
+      let withChildren = $('.cat-drop-stack ul li.has-children.parent-cat-wrap').find('ul.children.level-0');
+      withChildren.each(function(i, element) {
+        $(element).parent().find('a span.caret-icon').each(function(i, caret) {
+          caret.click();
+        });
+      });
+
       let selectedLi = $('.cat-drop-stack ul').find('a.selected');
       selectedLi.css({ fontWeight: 'bold' });
 
       selectedLi.parents('ul.children').each(function(i, val) {
         $(val).css({ display: 'block' });
       });
+
+      let ulArea = $( '.store-cat-stack-dokan.cat-drop-stack ul');
```

Comment:
can we move this CSS to corresponding less file?

Reply:
Moved as per your recommendation. Thanks for the suggestion.